node/src/Pos/Explorer/DB.hs

```diff
@@ -257,8 +414,18 @@ blockPagePrefix page = "e/page/" <> encodedPage
   where
     encodedPage = serialize' $ UnsignedVarInt page
 
-blockEpochPrefix :: Epoch -> ByteString
-blockEpochPrefix epoch = "e/epoch/" <> serialize' epoch
+-- | TODO(ks): To remove in the next version.
+previousEpochBlocksPrefix :: Epoch -> ByteString
+previousEpochBlocksPrefix epoch = "e/epoch/" <> serialize' epoch
```

Comment:
It'd be better to call this "old" rather than "previous". Previous makes one thing of the epoch before this one, rather than an older version of the DB key.

Reply:
Ok, will fix.